solution-3.txt

```diff
@@ -0,0 +1,26 @@
+bca_info = """BCA is bachlor computer application"""
+
+with open('bca.txt','w') as bca:
+    bca.write(bca_info)
+
+A = 0
+B = 0
+C = 0
+
+from csv import reader

```

Comment:
in text file , it is  compulsory to import any module?

Reply:
NO,because it is not compulsory.